guides/source/action_cable_overview.md

```diff
@@ -266,9 +266,9 @@ WebSocket is opened.
 createConsumer('https://ws.example.com/cable')
 
 // Use a function to dynamically generate the URL
-createConsumer(getWebSocketURL)
+createConsumer(getWebSocketURL())
```

Comment:
This was correct before - it's documenting that you can pass a function by reference.

https://github.com/rails/rails/blob/491352518034c2d825d8604a30c7c27932ae9cea/actioncable/app/javascript/action_cable/consumer.js#L61-L63

Reply:
I restored the previous version of this line in https://github.com/rails/rails/commit/44b6ea5942337e2c4f98d235b23e827dbbaf598f. Thanks for the other fix, though 🙂